guesttemplates/blank_template.py

```diff
@@ -1,3 +1,6 @@
+from builtins import str
+from past.builtins import basestring
+from builtins import object
```

Comment:
```python
from builtins import object, str
from past.builtins import basestring
```

Reply:
I chose to first commit exactly what `futurize` produced, as a reference point, and then to improve on it. I could apply this suggestion in a new commit but as all the lines are then removed, I'm not sure it's necessary.